hw3/01-characters.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>01 - Characters</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="styles.css" />
+    <script defer src="01-characters.js"></script>
+  </head>
+  <body>
+    <main>
+      <h1>Exercise 01 - Characters</h1>
+      <section class="d-flex flex-wrap container mx-auto"></section>
```

Comment:
Nice job on this part: way simpler and cleaner than what I came up with. You might consider setting the width directly to match the image from the requirements more closely.